000-local-env.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+if ( defined( 'VIP_GO_ENV' ) && VIP_GO_ENV ) {
```

Comment:
> Hmm... I think this is right. The idea is that it returns false if it's not a VIP Go Environment.

In a perfect world, there is little difference between running code on Go production and on N local environments. Those local environments should also be able to specify what their 'environmen name' is - like 'QA-12345' or 'nicks-local-docker-install'.

If we use this constant as specifying if it's running on Go hardware or not, we can't distinguish different local environments from one-another, and we already have a constant that was made to tell you if you are running on production hardware or not, which is `WPCOM_IS_VIP_ENV`.

We use that exact name for backwards compatibility with dotcom code, but it's still important to specify the _where_ (Go hardware vs other) and the _what_ (what 'version' of the site this is) separately.